src/components/sbb-loading-indicator/sbb-loading-indicator.scss

```diff
@@ -0,0 +1,235 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host([variant='circle']) {
+  --sbb-loading-indicator-height: var(--sbb-size-icon-ui-small);
+  --sbb-loading-indicator-width: var(--sbb-size-icon-ui-small);
+  --sbb-loading-indicator-padding: var(--sbb-border-width-2x);
+  --sbb-loader-indicator-duration: 1s;
+
+  display: inline-block;
+  line-height: 0;
+}
+
+:host([variant='circle']) .sbb-loading-indicator {
+  height: var(--sbb-loading-indicator-height);
+  width: var(--sbb-loading-indicator-width);
+  padding-inline: var(--sbb-loading-indicator-padding);
+  padding-block: var(--sbb-loading-indicator-padding);
+  display: inline-block;
+}
+
+:host([variant='circle']) .sbb-loading-indicator__animated-element {
+  width: 100%;
+  height: 100%;
+  display: inline-block;
+  color: transparent;
+  position: relative;
+  margin: 0 auto;
+  overflow: hidden;
+  border-radius: 50%;
+  background: conic-gradient(from 90deg, #fff, #f00);
+  /* stylelint-disable */
+  -webkit-mask: radial-gradient(circle 7px, #0000 98%, #fff);
+  /* stylelint-enable */
+  mask: radial-gradient(circle 7px, #0000 98%, #fff);
```

Comment:
Isn't it possible to use our tokens for the other colours too? (`sbb-color-black-default`, `sbb-color-white-default`...)

Reply:
color-white-default ok
the #0000 instead is for transparent in rgba

Reply:
Maybe you can try to use the token like this `sbb-tokens.$sbb-color-black-default`? (see `sbb-notification` component)